project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -1232,9 +1232,13 @@ function c4m_message_load_homepage_activity_stream() {
 
   if ($user->uid) {
     // This is an Authenticated user.
-    if ($filter == 'interests' || $filter == 'groups') {
+    if ($filter == 'interests' || $filter == 'groups' || $filter == 'projects') {
       $groups = c4m_og_get_user_group_nodes_by_bundle('group');
```

Comment:
@guytepper 
Let's use the `switch` as discussed yesterday so we won't hit the DB if not needed ...